com.sap.cloud.lm.sl.cf.process/src/main/java/com/sap/cloud/lm/sl/cf/process/jobs/CleanUpJob.java

```diff
@@ -115,7 +115,7 @@ private Date getExpirationTime() {
         return cleanUpTimestamp;
     }
 
-    private void abortOldOperationsInActiveState(Date expirationTime) {
+    void abortOldOperationsInActiveState(Date expirationTime) {

```

Comment:
I will not review this but please do not expose methods just to test them. If we need to do such thing - maybe something is not right. :)

Reply:
I've returned them to private. We might have some fake code coverage now because not all cleanup steps are covered in the test class.

Reply:
Yes, but returning the functions to private does not mean that they cannot be tested. Some refactoring could fix the things :) - nvm next time.

Reply:
The ones that were non-private are tested as I changed the test class to execute the parent public function. :)